test/functional/plugin/lsp_spec.lua

```diff
@@ -985,7 +985,7 @@ describe('LSP', function()
       }
       local completion_list_items = {items=completion_list}
       local expected = {
-        { abbr = 'foobar', dup = 1, empty = 1, icase = 1, info = ' ', kind = 'Unknown', menu = '', word = 'foobar', user_data = { nvim = { lsp = { completion_item = { label = 'foobar' } } } } },
+        { abbr = 'foobar', dup = 1, empty = 1, icase = 1, info = ' ', kind = 'Unknown', menu = '', word = 'foobar', user_data = { nvim = { lsp = { completion_item = { label='foobar' } } } } },
```

Comment:
nit: formatting change here

Reply:
I changed this to match all the other lines in that test. Should I revert?

Reply:
Bump. This is ready otherwise.